src/app/docs/paging/paging-docs.component.html

```diff
@@ -0,0 +1,20 @@
+<sky-docs-demo-page
+  moduleName="SkyPagingModule"
+  moduleSourceCodePath="src/app/public/modules/paging/"
+  pageTitle="Paging"
+>
+  <sky-docs-demo-page-summary>
+    The paging component displays a SKY UX-themed pagination control that allows you to display list data across multiple pages. If the data fits on a single page, the component hides the pagination control because it is not necessary.
```

Comment:
To follow the pattern of short intros, I'd tighten this up to `The paging component provides a SKY UX-themed pagination control to display list data across multiple pages.`

Reply:
Done.

Reply:
This update didn't make it in for some reason, but I went ahead and made it myself. Thanks.